psalm.xml

```diff
@@ -17,6 +17,7 @@
     </projectFiles>
 
     <issueHandlers>
+        <RedundantPropertyInitializationCheck errorLevel="suppress"/>
```

Comment:
Do we want to suppress this globally? Is this the "should be initialized in constructor" error or a different initialization error?

Reply:
It's similar to the "should be initialized in constructor" error. It's thrown for `isset()` checks on the properties which are initialized in the getters instead of constructors.

Reply:
If we plan to avoid the typed property initialization, then I agree it should just be suppressed. Wasn't sure the policy we wanted to adopt.

Reply:
In general initialising properties in getters instead of constructors is preferred at times. Also the psalm check/error is about redundancy, so I don't think we lose anything by suppressing this message.